patches/0034-Fix-tags_all-Computed-for-PF-resources.patch

```diff
@@ -18,18 +18,18 @@ index b6d8c71ef7..f78c4502b0 100644
  		Blocks: map[string]schema.Block{
  			names.AttrDestination: schema.ListNestedBlock{
 diff --git a/internal/service/bedrock/custom_model.go b/internal/service/bedrock/custom_model.go
-index e0606306fd..61981ae7fb 100644
+index daba31c85d..8ce47d7720 100644
 --- a/internal/service/bedrock/custom_model.go
 +++ b/internal/service/bedrock/custom_model.go
-@@ -137,7 +137,7 @@ func (r *customModelResource) Schema(ctx context.Context, request resource.Schem
+@@ -136,7 +136,7 @@ func (r *customModelResource) Schema(ctx context.Context, request resource.Schem
  				},
  			},
- 			names.AttrTags:    tftags.TagsAttribute(),
--			names.AttrTagsAll: tftags.TagsAttributeComputedOnly(),
-+			names.AttrTagsAll: tftags.TagsAttribute(),
- 			"training_metrics": schema.ListAttribute{
- 				CustomType: fwtypes.NewListNestedObjectTypeOf[customModelTrainingMetricsModel](ctx),
- 				Computed:   true,
+ 			names.AttrTags:       tftags.TagsAttribute(),
+-			names.AttrTagsAll:    tftags.TagsAttributeComputedOnly(),
++			names.AttrTagsAll:    tftags.TagsAttribute(),
```

Comment:
Patch conflict here. I think simply the surrounding code changed upstream, so all this does is makes sure the patch still cleanly applies, while the real edits are removing ComputedOnly.